### ISSUE TITLE
uses scientific notation when not necessary

### ISSUE THREAD

**Issue by goldmanm** [randlet/to-precision#1]
When inputting `to_precision(132,2)`, I would expect it to return `'130'`, but it actually gives `'1.3e+2'`. To my knowledge, both have the valid amount of significant figures, and the first one is easier to read. We might want to add a trailing zeros check. 

I tried it on python 2.7 and 3.4 and got the same result. 

Thanks so much for sharing this. I really like the code.

**Comment by randlet** [comment]
Yes you're right 130 is better there.   Maybe for anything less than 1000 (or 10000? 9300 vs 9.3e3) displaying the trailing zeros is better.

**Comment by goldmanm** [comment]
The [string formatting using the letter 'g'](https://docs.python.org/3/library/string.html#format-string-syntax) seems to have a default to not use exponential notation if the exponent is between -4 and some number 'p', which I could see also being around 4. 

If the significant figures can be used correctly (no significant trailing zeros), checking whether the value of the exponent is between 0 and 4 (or 3), and then displaying that using regular notation might be a way to do this.

**Comment by randlet** [comment]
Yes, I agree following the `g` formatting makes sense.

**Comment by BebeSparkelSparkel** [comment]
@goldmanm 
I rewrote some of this repo and anything > -1000 and < 1000 will be in standard notation otherwise it goes to scientific notation.

You can check it out at:
[github.com/BebeSparkelSparkel/to-precision](https://github.com/BebeSparkelSparkel/to-precision)
